.travis/build.sh

```diff
@@ -19,6 +19,12 @@ export DOCKER_ORG=${DOCKER_ORG:-strimzici}
 export DOCKER_REGISTRY=${DOCKER_REGISTRY:-docker.io}
 export DOCKER_TAG=$COMMIT
 
+API_URL=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
+API_TOKEN=$(kubectl describe secret $(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}') | grep -E '^token' | cut -f2 -d':' | tr -d " ")
```

Comment:
The default service account should normally have minimal privileges. I'm not sure I understand wat is the value of getting its token. Moreover AFAIK not every service account can actually authenticate from the outside. You anyway seem to redefine these things later, so I'm not sure what is the value of this change in `build.sh`.